apps/server/src/shared/repo/dashboard/dashboard.repo.integration.spec.ts

```diff
@@ -50,17 +51,21 @@ describe('dashboard repo', () => {
 	});
 
 	it('should persist dashboard with gridElement group', async () => {
+		const user = userFactory.build();
+		const firstcourse = courseFactory.build({ students: [user] });
+		const secondCourse = courseFactory.build({ students: [user] });
```

Comment:
to be consistent with the following tests I suggest using courseFactory.buildList here as well